docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -778,12 +778,48 @@ contract ExampleHelloWorld {
 
 Please note that this contract is simply a wrapper and is calling the precompile functions.
 
-### Step 7: Add Precompile Solidity Tests
+### Step 7: Building AvalancheGo and Subnet-EVM
 
-#### Step 7.1: Add Hardhat Test
+Before we start testing, we will need to build the AvalancheGo binary and the custom Subnet-EVM binary.
+
+You should have cloned [AvalancheGo](https://github.com/ava-labs/avalanchego) within your `$GOPATH` in the [Prerequisites](#prerequisites) section, so you can build AvalancheGo with the following command:
+
+```bash
+cd $GOPATH/src/github.com/ava-labs/avalanchego
+./scripts/build.sh
+```
+
+Once you've built AvalancheGo, you can confirm that it was successful by printing the version:
+
+```bash
+./build/avalanchego --version
+```
+
+This should print something like the following (if you are running AvalancheGo v1.9.7):
+
+```
+avalanche/1.9.7 [database=v1.4.5, rpcchainvm=22, commit=3e3e40f2f4658183d999807b724245023a13f5dc]
+```
+
+Once we've built AvalancheGo, we can navigate back to the Subnet-EVM repo and build the Subnet-EVM binary:
+
+```bash
+cd $GOPATH/src/github.com/ava-labs/subnet-evm
+./scripts/build.sh
+```
+
+This will build the Subnet-EVM binary and place it in AvalancheGo's `build/plugins` directory by default at the filepath:
```

Comment:
Either change `filepath` to `file path` or add `filepath` to `.github/styles/Vocab/Products/accept.txt`